### ISSUE TITLE
styled-components > memoize-one@4.1.0: New custom equality api does not play well with all equality helpers. Please use v5.x

### ISSUE THREAD

**Issue by basslagter** [styled-components/styled-components#2311]
When installing (using Yarn) the following warning is shown:

styled-components > memoize-one@4.1.0: New custom equality api does not play well with all equality helpers. Please use v5.x

**Comment by bobirpulatov** [comment]
I have the same error, plz help!

**Comment by mxstbr** [comment]
A PR figuring out where this is coming from and upgrading us to the new version would be appreciated! :pray:

**Comment by adam187** [comment]
@mxstbr this commit should solve issue: https://github.com/styled-components/styled-components/commit/2d82900867bd9bd95bf6ba9b516cb648456084be

**Comment by adam187** [comment]
@mxstbr this commit should solve issue: https://github.com/styled-components/styled-components/commit/2d82900867bd9bd95bf6ba9b516cb648456084be but when you install `styled-components@4.1.3` you still get `memoize-one@4.1.0`

```
# yarn.lock

styled-components@4.1.3:
  version "4.1.3"
  resolved "https://registry.yarnpkg.com/styled-components/-/styled-components-4.1.3.tgz#4472447208e618b57e84deaaeb6acd34a5e0fe9b"
  integrity sha512-0quV4KnSfvq5iMtT0RzpMGl/Dg3XIxIxOl9eJpiqiq4SrAmR1l1DLzNpMzoy3DyzdXVDMJS2HzROnXscWA3SEw==
  dependencies:
    "@babel/helper-module-imports" "^7.0.0"
    "@emotion/is-prop-valid" "^0.7.3"
    "@emotion/unitless" "^0.7.0"
    babel-plugin-styled-components ">= 1"
    css-to-react-native "^2.2.2"
    memoize-one "^4.0.0"  <----------
    prop-types "^15.5.4"
    react-is "^16.6.0"
    stylis "^3.5.0"
    stylis-rule-sheet "^0.0.10"
    supports-color "^5.5.0"
```

```
# package-lock.json

    "styled-components": {
      "version": "4.1.3",
      "resolved": "https://registry.npmjs.org/styled-components/-/styled-components-4.1.3.tgz",
      "integrity": "sha512-0quV4KnSfvq5iMtT0RzpMGl/Dg3XIxIxOl9eJpiqiq4SrAmR1l1DLzNpMzoy3DyzdXVDMJS2HzROnXscWA3SEw==",
      "requires": {
        "@babel/helper-module-imports": "7.0.0",
        "@emotion/is-prop-valid": "0.7.3",
        "@emotion/unitless": "0.7.3",
        "babel-plugin-styled-components": "1.10.0",
        "css-to-react-native": "2.2.2",
        "memoize-one": "4.1.0", <----------
        "prop-types": "15.6.2",
        "react-is": "16.7.0",
        "stylis": "3.5.4",
        "stylis-rule-sheet": "0.0.10",
        "supports-color": "5.5.0"
      }
    },
```

 and according to this commit https://github.com/styled-components/styled-components/commit/8cf278bf4e3476b1f5f75006e90b81ad86bccdff it should already be in 4.1.3 but it isn't. 

So probably something went wrong with `4.1.3` release. 

Also there is no tag for `4.1.3`
<img width="299" alt="screenshot 2019-01-19 at 13 12 46" src="https://user-images.githubusercontent.com/156628/51426494-f993fa80-1beb-11e9-9b72-159237061576.png">

**Comment by mxstbr** [comment]
Can you try `styled-components@4.1.4-alpha.2`?

**Comment by basslagter** [comment]
@mxstbr Seems fixed!

**Comment by mxstbr** [comment]
Cool, this will be fixed in v4.1.4 then! Closing since that'll be out soon :100:

**Comment by basslagter** [comment]
Awesome! Thanks for the response!
